### ISSUE TITLE
Cannot set fee rate of 249 ppm

### ISSUE THREAD

**Issue by cryptosharks131** [lightningnetwork/lnd#6189]
### Background

When setting a fee rate of 249 ppm on a channel, the value is set to 248 ppm instead

### Your environment

* lnd v0.14.1-beta

### Steps to reproduce

Set a channel to a fee rate of 249 and then query lnd for the fee rate policy on that channel

### Expected behaviour

Fee rate becomes 249

### Actual behaviour

Fee rate becomes 248

**Comment by guggero** [comment]
This is very weird. I see no reason why the value should be changed. Is it possible you previously set the value to 248 ppm and the update to 249 just silently _failed_?

**Comment by cryptosharks131** [comment]
Never used a fee rate of 249 ppm before and only tried it after a user reported it as a bug in lndg.

After investigating, I was able to easily reproduce the issue on all my lnd instances and using both gRPC and lncli directly.

**Comment by guggero** [comment]
Okay, very interesting! Thanks for confirming.

**Comment by niteshbalusu11** [comment]
It also reduces the fee rate by 1ppm for 4004 and 4003.

**Comment by bhandras** [comment]
I believe this is due to a conversion bug that's been fixed but PR is currently waiting on the author. We can cherry pick it to a new PR too. The PR: https://github.com/lightningnetwork/lnd/pull/5711/

**Comment by alexbosworth** [comment]
Same issue as https://github.com/lightningnetwork/lnd/issues/4515 ?

**Comment by bhandras** [comment]
> Same issue as #4515 ?

Yes.
